docs/root/configuration/statistics.rst

```diff
@@ -16,7 +16,7 @@ Server related statistics are rooted at *server.* with following statistics:
   concurrency, Gauge, Number of worker threads
   memory_allocated, Gauge, Current amount of allocated memory in bytes. Total of both new and old Envoy processes on hot restart. 
   memory_heap_size, Gauge, Current reserved heap size in bytes. New Envoy process heap size on hot restart. 
-  live, Gauge, "1 if the server is not currently draining, 0 otherwise"
+  live, Gauge, Current :ref:`State <envoy_api_enum_admin.v2alpha.ServerInfo.state>` of the Server.
```

Comment:
`live` is not a perfect name here - Thought of introducing another stat called `state` - but decided to enhance this. If any one have objections, I can introduce a new stat - but that may create confusion. We can not change the name because it is very widely used.

Reply:
On a second thought decided to add a separate stat to avoid confusion while it may be little overlapping with `server.live` stat. Just changed the enum values so that they align with what `server.live` shows for `LIVE` and `DRAINING`.

I am open to any other better option.